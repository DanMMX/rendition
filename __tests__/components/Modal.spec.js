import React from 'react';
import renderer from 'react-test-renderer';
import Provider from '../../src/components/Provider'
import Modal from '../../src/components/Modal'

test('Modal renders correctly', () => {
  const component = renderer.create(
    <Provider>
      <Modal>Modal</Modal>
    </Provider>,
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
