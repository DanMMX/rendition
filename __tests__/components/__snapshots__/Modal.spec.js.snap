// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Modal renders correctly 1`] = `
<div
  className="sc-bdVaJa jKopLr"
>
  <div
    className="sc-hMqMXs gOARcJ sc-chPdSV dfDVBP sc-cSHVUG kBGaNN sc-jzJRlG gnRVez"
    onClick={[Function]}
  >
    <div
      className="sc-kEYyzF gMkvag sc-jTzLTM gkrlrR sc-VigVT gYpBDW"
      z={8888}
    />
    <div
      className="sc-hSdWYo QyOcr sc-kgoBCf jIcSzk sc-kAzzGY bHluuy sc-fjdhpX jWzEVr"
    >
      <div
        className="sc-eHgmQL bWRhNx sc-kgoBCf cUYCEa sc-kAzzGY bHluuy sc-fjdhpX gjXQTW"
        onClick={[Function]}
      >
        Modal
        <div
          className="sc-chPdSV WNwZE sc-cSHVUG kBGaNN sc-jzJRlG gBjLKB"
        >
          <button
            className="sc-gqjmRU bCjhLA sc-htoDjs hfajhw"
            onClick={undefined}
          >
            OK
          </button>
        </div>
      </div>
    </div>
  </div>
</div>
`;
